### DISCUSSION TITLE
Sort group

### DISCUSSION THREAD

**Discussion by sylvesterdamgaard** [directus/directus#16167] [Archive]
Is it possible to scope the sorting to another column?
E.g. i have an election card model which have parties related to it and i have a candidates related to the parties.

I which to be able to sort election cards on a global level. This is the default behavior, so all good.
I then wish to sort the parties, but in scope of the election card. 
And again, the candidates should be sorted in the scope of the party.

Here's a screenshot from a custom app where we can drag/drop the order with the structure described above.
![image](https://user-images.githubusercontent.com/2431914/197808595-49034a31-61f4-4755-99c0-829a0ac0872a.png)

For a Laravel application we use this package to help us achieve this.
https://github.com/spatie/eloquent-sortable

**Comment by rijkvanzanten** [comment]
I think that should be possible by using a many to many with a sort field for both the election-cards to parties and parties to party-members 🙂

**Reply by sylvesterdamgaard** [reply]
A bit of a workaround, but i will give it a go and see if there's any tradeoffs. Thank you. :)
